mara_storage/client.py

```diff
@@ -55,3 +55,8 @@ def __(storage: storages.LocalStorage):
 def __(storage: storages.GoogleCloudStorage):
     from .google_cloud_storage import GoogleCloudStorageClient
     return GoogleCloudStorageClient
+
```

Comment:
Shouldn't this be in the azure file?

Reply:
Oh wait, that would mean there end user would need to import it, prob not a good idea

Reply:
Right. But I like the idea to rethink the whole storage client. Maybe #12 could help here …